controls/map/styling/shapesstyles.md

```diff
@@ -18,33 +18,33 @@ The Map `ShapefileLayer` exposes `ShapeStyle` property which helps you to style
 * `StrokeColor`
 * `FillColor`
 
-The snippet below shows how ShapeStyle property can be applied:
+The snippet below shows how `ShapeStyle` property can be applied:
 
 <snippet id='map-styling-shapeslstyle' />
 
 Here is the result:
 
-![Map Shapes Styling](../images/map_styling_shapestyle.png)
+![.NET MAUI Map Shapes Styling](../images/map_styling_shapestyle.png)
 
-where the `Source` and the `DataSource` of the MapShapeReader should be set to a .shp and .dbf files, respectively:
+Define the `Source` and the `DataSource` of the `MapShapeReader` to a `.shp` and `.dbf` files, respectively:
 
 <snippet id='map-interactionmode-settintsource' />
 
 ## Style Selector for Shapes
 
-Using the `ShapeStyleSelector` property of the `ShapefileLayer` you could implement conditional styling. 
+Using the `ShapeStyleSelector` property of the `ShapefileLayer` you can implement conditional styling. 
 
 The example below shows how to apply different styles to shapes according to certain property value of each shape.
 
 1. First, create the selector class which inherit from `MapShapeStyleSelector`:
```

Comment:
use bold list here: 
**1.**
**2.**, etc

and remove the space before the snippet id